ADR/Components/Aerodynamic_components/VS.py

```diff
@@ -7,8 +7,6 @@ class VS(Aerodynamic_surface):
     def __init__(self, data):
         super().__init__(data)
 
-        self.utilidade_do_EV = "Nenhuma irmao, sai dessa vida!"
-
         self.calc_area()
```

Comment:
Although a good modification, this doesn't belong to the Horizontal Stabilizer class, so it would be good to move this modification to a commit on another PR (one for the Vertical Stabilizer class).

Reply:
Now I see that this PR will develop tests for several components, so just change the title adding VS and Wing.